addons/OPT/modules.hpp

```diff
@@ -22,6 +22,7 @@ class CfgCLibModules {
             FNC(clientinitEH);
             FNC(getUnitIconID);
             FNC(updateUnitIcon);
+			FNC(miniGPS);			
```

Comment:
Einrückung passt nicht zu den vorherigen Einträgen

Reply:
habe das noch mal neu eingerückt, in VSC ist das alles richte bei mir